torch/nn/modules/pooling.py

```diff
@@ -739,9 +739,12 @@ class FractionalMaxPool2d(Module):
         kernel_size: the size of the window to take a max over.
                      Can be a single number k (for a square kernel of k x k) or a tuple `(kh, kw)`
         output_size: the target output size of the image of the form `oH x oW`.
-                     Can be a tuple `(oH, oW)` or a single number oH for a square image `oH x oH`
+                     Can be a tuple `(oH, oW)` or a single number oH for a square image `oH x oH`.
+                     Note that we must have :math:`kH + oH -1 <= H_{in}` and :math:`kW + oW - 1 <= W_{in}`
         output_ratio: If one wants to have an output size as a ratio of the input size, this option can be given.
-                      This has to be a number or tuple in the range (0, 1)
+                      This has to be a number or tuple in the range (0, 1).
+                      Note that we must have :math:`kH + (output_ratio_H * H_{in}) - 1 <= H_{in}`
+                      and :math:`kW + (output_ratio_W * W_{in}) - 1 <= W_{in}`
```

Comment:
Oops, looks like the rendering is a bit off for this 
<img width="731" alt="Screenshot 2023-12-27 at 12 37 39 PM" src="https://github.com/pytorch/pytorch/assets/35276741/d2419989-f50b-4feb-b697-52875de77e4e">

```suggestion
                      Note that we must have :math:`kH + (output\_ratio_H * H_{in}) - 1 <= H_{in}`
                      and :math:`kW + (output\_ratio\_W * W_{in}) - 1 <= W_{in}`
```

For future reference you can check the rendering in the [pytorchbot comment on the PR](https://github.com/pytorch/pytorch/pull/116261#issuecomment-1866075623) under 

> 📄 Preview [Python docs built from this PR](https://docs-preview.pytorch.org/pytorch/pytorch/116261/index.html)